src/json.hpp.re2c

```diff
@@ -6168,6 +6168,21 @@ class basic_json
               thousands_sep(!loc->thousands_sep ? '\0' : loc->thousands_sep[0]),
               decimal_point(!loc->decimal_point ? '\0' : loc->decimal_point[0])
         {}
+        serializer(const serializer& cpy)
```

Comment:
I agree that there is no need to copy it in the first place, since this is like a traits struct, with no attributes.

I didn't look at the diff, but if we need to add explicit constructors on stuff like `detail::is_nested_class`, just to support this warning, I really think this isn't worth it, the tests do not build with it, and I believe people who want to use it in their codebase should include `json.hpp` with `-isystem`